### PR TITLE
chore: miscellaneous changes

### PR DESCRIPTION
**Pull request by thiagowfx**

### CODE REVIEW

File: .github/workflows/publish.yml [hunk 1]
```diff
@@ -1,33 +1,33 @@
...
     name: Publish
     runs-on: ubuntu-latest
     steps:
-      - name: Check out repository
-        uses: actions/checkout@96f53100ba2a5449eb71d2e6604bbcd94b9449b5
-      - name: Set up Node.js
-        uses: actions/setup-node@698d50532ea4cc0490274810a45fe52318110a1c
+      - name: Check out repo
+        uses: actions/checkout@v3
+      - name: Set up node.js
```
Comment (OrKoN) [inline .github/workflows/publish.yml:19]: node.js is correctly spelled as 'Node.js'

File: .github/workflows/publish.yml [hunk 2]
```diff
@@ -1,33 +1,33 @@
...
   publish:
     name: Publish
     runs-on: ubuntu-latest
     steps:
-      - name: Check out repository
-        uses: actions/checkout@96f53100ba2a5449eb71d2e6604bbcd94b9449b5
-      - name: Set up Node.js
-        uses: actions/setup-node@698d50532ea4cc0490274810a45fe52318110a1c
+      - name: Check out repo
+        uses: actions/checkout@v3
```
Comment (OrKoN) [inline .github/workflows/publish.yml:18]: using hashes is a security recommendation. Put the version as a comment next to the hash.

Comment (thiagowfx) [reply .github/workflows/publish.yml:18]: Better to address this one separately. Lots of manual work to get each individual hash. @jrandolf is there a way to automate this process?

Comment (OrKoN) [reply .github/workflows/publish.yml:18]: we could copy the hashes from this workflow (or perhaps from Puppeteer) and then let dependabot do the updates

Comment (OrKoN) [reply .github/workflows/publish.yml:18]: e.g., https://github.com/puppeteer/puppeteer/blob/main/.github/workflows/publish.yml#L19

Comment (OrKoN) [reply .github/workflows/publish.yml:18]: btw I counted only four unique packages so updating manually (find & replace) should not be too problematic

Comment (thiagowfx) [reply .github/workflows/publish.yml]: There are way more than four, updating them is quite tedious

**Review by OrKoN — APPROVED** [review]

**Pull request merged**